pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -1292,7 +1292,7 @@ func buildDefaultCluster(push *model.PushContext, name string, discoveryType api
 		cluster.DnsLookupFamily = apiv2.Cluster_V4_ONLY
 		dnsRate := gogo.DurationToProtoDuration(push.Mesh.DnsRefreshRate)
 		cluster.DnsRefreshRate = dnsRate
-		if util.IsIstioVersionGE13(proxy) && features.RespectDNSTTL.Get() {
+		if util.IsIstioVersionGE13(proxy) {
```

Comment:
Do we still need this condition?

Reply:
This is another question, so in 1.16 this kind of checks can be removed.  Just as we did before, maybe in another pr, remove all of them.